k8s/overlays/coreso/prod/ingress-gridcapa.yaml

```diff
@@ -348,6 +348,325 @@ spec:
 ---
 apiVersion: networking.k8s.io/v1
 kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: kommander-traefik
+    traefik.ingress.kubernetes.io/router.middlewares: default-gridcapa-swe-d2cc@kubernetescrd,default-gridcapa-user-id@kubernetescrd
+    traefik.ingress.kubernetes.io/router.tls: "true"
+  labels:
+    app: gridcapa
+    component: gridcapa-ingress
+  name: gridcapa-ingress-swe-d2cc-app
+  namespace: default
+spec:
+  rules:
+    - host: gridcapa.ingress-gridcapa-test-vaas.rh.1648.local
```

Comment:
The host for prod should be gridcapa.ingress-gridcapa-vaas.rh.1648.local for all ingress added in this file